misc/python/materialize/checks/scenarios_upgrade.py

```diff
@@ -88,7 +88,7 @@ class UpgradeEntireMzFromLatestLTS(Scenario):
     """Upgrade the entire Mz instance from the last LTS version without any intermediate steps. This makes sure our LTS releases for self-managed Materialize stay upgradable."""
 
     def base_version(self) -> MzVersion:
-        return LTS_VERSIONS[-1]
+        return get_lts_versions()[-1]
```

Comment:
This is great!

Will this pickup pre-released versions we're actively validating as well? Do we want it to if it does?